docs/connectors/slack.md

```diff
@@ -19,34 +26,65 @@ connectors:
     # required
     token: "xoxb-abdcefghi-12345"
     # optional
-    user-token: "xoxp-abcdef-123" # default None. ** 
-    bot-name: "mybot" # default "opsdroid"
+    socket-mode: True # defaul false. if true app-token is required
```

Comment:
Is there any use for the app-token if socket mode isn't being used?

I am of a mind to suggest that we default to `socket-mode: True`, as that's closer to the current behaviour with RTM and `socket-mode: False` needs extra configuration to make the API endpoint publicly accessible.

Reply:
socket-mode now defaults to True